compiler/rustc_parse/src/parser/ty.rs

```diff
@@ -702,7 +702,7 @@ impl<'a> Parser<'a> {
     /// Is a `dyn B0 + ... + Bn` type allowed here?
     fn is_explicit_dyn_type(&mut self) -> bool {
         self.check_keyword(kw::Dyn)
-            && (!self.token.uninterpolated_span().rust_2015()
+            && (self.token.uninterpolated_span().rust_2018()
```

Comment:
Wouldn't it make sense for this similarly to be renamed `is_rust_2018`, and so on for all such edition tests?

Reply:
No, see the linked comment. `rust_2015` is `== 2015`, but `rust_2018` is `>= 2018` (similarly for others).